build/package/send-slack-notification.sh

```diff
@@ -15,11 +15,6 @@
 
 set -Eeou pipefail
 
-if [[ "${unstable-}" == "-unstable" ]]; then
```

Comment:
[q] for my own learning what cases do we have `unstable-`?

Reply:
package on merge master, otherwise normal release